Organization.md

```diff
@@ -0,0 +1,102 @@
+# Organization
+
+Currently, the Distributed Text Services is not legally formalized as an organization (outside of the Github Organization and the Google Group).
+
+Although, the Distributed Text Services specifications is lead by a [Technical Committee](#technical_committee). It is responsible for the evolution and documentation of the specifications.
+
+
+## Technical Committee
+
+* The **Technical committee** is responsible for voting on feature requests and bugs
+* Once the initial committee is established, the committee itself is responsible for approving new members to the committee
+*  New feature requests must be voted on by the technical committee in order to be approved as additions to the spec.
+* Committee members have 1 month after a feature request is officially opened for voting to provide their vote.  If you don't vote within that time frame, it is the same as voting yes. The exception is feature requests opened for voting in July and August. The voting period for these is extended to 2 months.
+* Decisions are normally made by consensus.  If there is need for a vote, the chair
+is responsible for determining a fair process and ensuring adequately informed
+discussion ahead of time.
+* Consensus means that there is adequate representation, nobody cannot live with
+the decision, and there is general approval for the decision.
+
+## Roles
+
+### Technical Committee Member
+
+* attends at least 1/2 of the regular teleconferences
+* is responsible for voting on new feature requests and bugs
+* must take into account issues raised by observers and users of the standard
+* is responsible for contributing to development of specification features and bug fixes
+
+### Chair of the Technical Committee
+
+The chair is responsible to run the meetings, create agendas, ensure that issues are adequately understood before decisions are made, and seek decisions by consensus whenever possible. The chair also ensures that decisions are adequately recorded and assigns and tracks action items.
+
+### Ambassador
+
+* distinguishes important contributor to the community.
```

Comment:
Would recommend rewording this to "a member of the community who makes material contributions to the effort"